pylutron_caseta/smartbridge.py

```diff
@@ -39,6 +39,8 @@ def __init__(self, hostname=None):
         monitor.start()
         for _id in self.devices:
             self.get_value(_id)
+
+        self.set_value('41', 0)
```

Comment:
I'm a complete newcomer to this but I think this line could definitely use some comment